public/app/features/explore/RichHistory/RichHistory.tsx

```diff
@@ -45,141 +45,122 @@ export interface RichHistoryProps extends Themeable2 {
   onClose: () => void;
 }
 
-type RichHistoryState = {
-  loading: boolean;
-};
-
-class UnThemedRichHistory extends PureComponent<RichHistoryProps> {
-  state: RichHistoryState = {
-    loading: false,
-  };
-
-  updateSettings = (settingsToUpdate: Partial<RichHistorySettings>) => {
-    this.props.updateHistorySettings({ ...this.props.richHistorySettings, ...settingsToUpdate });
+function UnThemedRichHistory(props: RichHistoryProps) {
```

Comment:
AFAICT we do not use the theme anymore in here, we can simply export `RichHistory` and remove `extends Themeable2` from the props interface + `export const RichHistory = withTheme2(UnThemedRichHistory);` at the end of the file

```suggestion
export function RichHistory(props: RichHistoryProps) {
```